src/main/scala/strawman/collection/mutable/ArrayDeque.scala

```diff
@@ -0,0 +1,349 @@
+package scala                     //TODO: Change this strawman --> we just need sizeHintIfCheap from scala
```

Comment:
I think knownSize would work. QQ: Why is it returning -1 instead of Option[Int]? Perf?